features/kinderen.feature

```diff
@@ -1,71 +1,73 @@
 # language: nl
 
 Functionaliteit: Kinderen van een ingeschreven persoon raadplegen
-  Van een inschreven persoon kunnen kinderen worden opgevraagd als sub-resource van de ingeschreven persoon. De sub-resource kinderen bevat de gegevens over de ouder-kindrelatie inclusief enkele eigenschappen van het betreffende kind.
+  Van een inschreven persoon worden -indien gevraagd met de fields parameter- het kinderen geleverd.
+  Dit bevat enkele identificerende eigenschappen van het kinderen.
 
-  Wanneer het kind een ingeschreven persoon is, levert de sub-resource kinderen de actuele gegevens van het kind zoals die op de PL van het betreffende kind staan (dus NIET zoals ze in categorie 09/59 Kind staan).
+  Rule: de actuele gegevens van kinderen worden geleverd
 
-  Scenario: de ingeschreven persoon heeft geen kind
-    Gegeven de te raadplegen persoon heeft geen (geregistreerde) kinderen
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999011
-    Dan is het aantal gevonden kinderen 0
+    Scenario: kindgegevens gecorrigeerd
```

Comment:
Wat wordt bedoeld met kind gecorrigeerd?

Reply:
ik wijzig dit in "de naam van een kind is gecorrigeerd"